app/app.js

```diff
@@ -34,18 +34,7 @@ const app = express();
 app.use(compression());
 app.use(cors(DEFAULTCORS));
 app.use(express.urlencoded({ extended: true }));
-app.use(
-  helmet({
-    contentSecurityPolicy: {
-      directives: {
-        'default-src': [
-          "'self'", // eslint-disable-line
-          new URL(config.get('keycloak.serverUrl')).origin
-        ]
-      }
```

Comment:
this error shows up in console whenever you visit `api/`, `/api/v1` and /api/v1/docs'.
![image](https://github.com/bcgov/common-object-management-service/assets/58443522/fc1c8f86-6a0b-4bcf-81ed-e7e506ff3e11)
i wonder if adding `'img-src': ['data:', 'favico.ico'],` here would fix this